src/cmd_line.rs

```diff
@@ -109,9 +109,9 @@ pub fn handle_command_line_arguments(args: Vec<String>) -> Result<(), String> {
                 .help("Maximize the window"),
         )
         .arg(
-            Arg::with_name("multi_grid")
-                .long("multigrid")
-                .help("Enable Multigrid"),
+            Arg::with_name("no_multi_grid")
+                .long("nomultigrid")
```

Comment:
```suggestion
                .long("no-multigrid")
```

Maybe this is better choice

Reply:
I debated here, but there were other existing settings which concatted noidle so I went with this. Instead I think I'll add a dash to the other items

Reply:
Done